callstats-java-sdk/src/main/java/io/callstats/sdk/httpclient/CallStatsAsyncHttpClient.java

```diff
@@ -42,19 +44,23 @@ public CallStatsAsyncHttpClient() {
 		Properties prop = new Properties();
 		InputStream input = null;
 	     
-		input = getClass().getClassLoader().getResourceAsStream(CallStatsConst.CallStatsJavaSDKPropertyFileName);
-		if (input !=null){
-			try {
+		//input = getClass().getClassLoader().getResourceAsStream(CallStatsConst.CallStatsJavaSDKPropertyFileName);
+		
+		try {
+			input = new FileInputStream(CallStatsConst.CallStatsJavaSDKPropertyFileName);
+			if (input !=null){
 				prop.load(input);
 				BASE_URL = prop.getProperty("CallStats.BaseURL");
 				connectionTimeOut = Integer.parseInt(prop.getProperty("CallStats.ConnectionTimeOut"));
```

Comment:
Do we want some kin of default values if the keys do not exist? (e.g. I'm pretty this would currently throw an exception if the CallStats.ConnectionTimeOut key doesn't exist)